src/main/java/com/cloudbees/jenkins/plugins/sshagent/SSHAgentStep.java

```diff
@@ -0,0 +1,84 @@
+package com.cloudbees.jenkins.plugins.sshagent;
+
+import hudson.Extension;
+import org.jenkinsci.plugins.workflow.steps.AbstractStepDescriptorImpl;
+import org.jenkinsci.plugins.workflow.steps.AbstractStepImpl;
+import org.kohsuke.stapler.DataBoundConstructor;
+import org.kohsuke.stapler.DataBoundSetter;
+
+import java.io.Serializable;
+import java.util.List;
+
+public class SSHAgentStep extends AbstractStepImpl implements Serializable {
+
+    /**
+     * Serial Version UID.
+     */
+    private static final long serialVersionUID = 1641672112447457198L;
+
+    /**
+     * The {@link com.cloudbees.plugins.credentials.common.StandardUsernameCredentials#getId()}s of the credentials
+     * to use.
+     */
+    @DataBoundSetter
```

Comment:
@jglick It was working fine but if you think that there is a best style, go ahead.